### ISSUE TITLE
Webpack build error on Next.js 13.2

### ISSUE THREAD

**Issue by re-taro** [vanilla-extract-css/vanilla-extract#1013]
### Describe the bug

Error with internal webpack configuration at build time when using `@vanilla-extract/next-plugin` in Next.js v13.2 delivered yesterday.

### Reproduction

https://github.com/re-taro-dev/card.re-taro.dev

### System Info

```shell
System:
    OS: Windows 10 10.0.22621
    CPU: (12) x64 Intel(R) Core(TM) i7-10750H CPU @ 2.60GHz
    Memory: 3.98 GB / 15.79 GB
  Binaries:
    Node: 18.14.2 - ~\scoop\apps\nodejs-lts\current\node.EXE
    Yarn: 1.22.19 - ~\scoop\apps\yarn\current\bin\yarn.CMD
    npm: 9.5.0 - ~\scoop\apps\nodejs-lts\current\npm.CMD
  Browsers:
    Edge: Spartan (44.22621.1265.0), Chromium (110.0.1587.46)
    Internet Explorer: 11.0.22621.1
  npmPackages:
    @vanilla-extract/css: 1.9.5 => 1.9.5
    @vanilla-extract/next-plugin: 2.1.1 => 2.1.1
    webpack: 5.75.0 => 5.75.0
```

### Used Package Manager

pnpm

### Logs

```shell
Failed to compile.

./node_modules/.pnpm/@vanilla-extract+webpack-plugin@2.2.0_webpack@5.75.0/node_modules/@vanilla-extract/webpack-plugin/extracted.js.webpack[javascript/auto]!=!./node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/build/webpack/loaders/css-loader/src/index.js??ruleSet[1].rules[4].oneOf[0].use[1]!./node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/build/webpack/loaders/postcss-loader/src/index.js??ruleSet[1].rules[4].oneOf[0].use[2]!./node_modules/.pnpm/@vanilla-extract+webpack-plugin@2.2.0_webpack@5.75.0/node_modules/@vanilla-extract/webpack-plugin/virtualFileLoader/dist/vanilla-extract-webpack-plugin-virtualFileLoader.cjs.js?{"fileName":"src/styles/home.css.ts.vanilla.css","source":"QGtleWZyYW1lcyBfMW84d2RzdzIgewogIGZyb20gewogICAgb3BhY2l0eTogMDsKICB9CiAgdG8gewogICAgb3BhY2l0eTogMTsKICB9Cn0KLl8xbzh3ZHN3MCB7CiAgd2lkdGg6IDEwMHZ3OwogIGhlaWdodDogMTAwdmg7CiAgcG9zaXRpb246IGFic29sdXRlOwogIHRvcDogMDsKICBsZWZ0OiAwOwogIHotaW5kZXg6IDA7Cn0KLl8xbzh3ZHN3MSB7CiAgcG9zaXRpb246IGFic29sdXRlOwogIHRvcDogMC40cmVtOwogIHJpZ2h0OiAwLjVyZW07CiAgZm9udC1zaXplOiAxcmVtOwogIG9wYWNpdHk6IDAuNDsKfQouXzFvOHdkc3czIHsKICBkaXNwbGF5OiBpbmxpbmUtZmxleDsKICBqdXN0aWZ5LWNvbnRlbnQ6IGNlbnRlcjsKICBhbGlnbi1pdGVtczogY2VudGVyOwogIGNvbG9yOiAjNGM1NjZhOwogIG1hcmdpbjogMC4ycmVtIDAuMnJlbSAwIDAuNXJlbTsKICBnYXA6IDAuNXJlbTsKICBhbmltYXRpb246IF8xbzh3ZHN3MiAycyBlYXNlLW91dCAxcyBib3RoOwogIHotaW5kZXg6IDEwMDsKfQouXzFvOHdkc3c0IHsKICBmb250LXNpemU6IDEuMnJlbTsKICBmb250LXdlaWdodDogMzAwOwogIHotaW5kZXg6IDA7Cn0="}!./node_modules/.pnpm/@vanilla-extract+webpack-plugin@2.2.0_webpack@5.75.0/node_modules/@vanilla-extract/webpack-plugin/extracted.js
TypeError: options.postcss is not a function
    at /home/runner/work/card.re-taro.dev/card.re-taro.dev/node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/build/webpack/loaders/postcss-loader/src/index.js:37:55
    at Span.traceAsyncFn (/home/runner/work/card.re-taro.dev/card.re-taro.dev/node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/trace/trace.js:79:26)
    at Object.loader (/home/runner/work/card.re-taro.dev/card.re-taro.dev/node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/build/webpack/loaders/postcss-loader/src/index.js:12:16)

Import trace for requested module:
./node_modules/.pnpm/@vanilla-extract+webpack-plugin@2.2.0_webpack@5.75.0/node_modules/@vanilla-extract/webpack-plugin/extracted.js.webpack[javascript/auto]!=!./node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/build/webpack/loaders/css-loader/src/index.js??ruleSet[1].rules[4].oneOf[0].use[1]!./node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/build/webpack/loaders/postcss-loader/src/index.js??ruleSet[1].rules[4].oneOf[0].use[2]!./node_modules/.pnpm/@vanilla-extract+webpack-plugin@2.2.0_webpack@5.75.0/node_modules/@vanilla-extract/webpack-plugin/virtualFileLoader/dist/vanilla-extract-webpack-plugin-virtualFileLoader.cjs.js?{"fileName":"src/styles/home.css.ts.vanilla.css","source":"QGtleWZyYW1lcyBfMW84d2RzdzIgewogIGZyb20gewogICAgb3BhY2l0eTogMDsKICB9CiAgdG8gewogICAgb3BhY2l0eTogMTsKICB9Cn0KLl8xbzh3ZHN3MCB7CiAgd2lkdGg6IDEwMHZ3OwogIGhlaWdodDogMTAwdmg7CiAgcG9zaXRpb246IGFic29sdXRlOwogIHRvcDogMDsKICBsZWZ0OiAwOwogIHotaW5kZXg6IDA7Cn0KLl8xbzh3ZHN3MSB7CiAgcG9zaXRpb246IGFic29sdXRlOwogIHRvcDogMC40cmVtOwogIHJpZ2h0OiAwLjVyZW07CiAgZm9udC1zaXplOiAxcmVtOwogIG9wYWNpdHk6IDAuNDsKfQouXzFvOHdkc3czIHsKICBkaXNwbGF5OiBpbmxpbmUtZmxleDsKICBqdXN0aWZ5LWNvbnRlbnQ6IGNlbnRlcjsKICBhbGlnbi1pdGVtczogY2VudGVyOwogIGNvbG9yOiAjNGM1NjZhOwogIG1hcmdpbjogMC4ycmVtIDAuMnJlbSAwIDAuNXJlbTsKICBnYXA6IDAuNXJlbTsKICBhbmltYXRpb246IF8xbzh3ZHN3MiAycyBlYXNlLW91dCAxcyBib3RoOwogIHotaW5kZXg6IDEwMDsKfQouXzFvOHdkc3c0IHsKICBmb250LXNpemU6IDEuMnJlbTsKICBmb250LXdlaWdodDogMzAwOwogIHotaW5kZXg6IDA7Cn0="}!./node_modules/.pnpm/@vanilla-extract+webpack-plugin@2.2.0_webpack@5.75.0/node_modules/@vanilla-extract/webpack-plugin/extracted.js
src/styles/home.css.ts.vanilla.css!=!./node_modules/.pnpm/@vanilla-extract+webpack-plugin@2.2.0_webpack@5.75.0/node_modules/@vanilla-extract/webpack-plugin/virtualFileLoader/dist/vanilla-extract-webpack-plugin-virtualFileLoader.cjs.js?{"fileName":"src/styles/home.css.ts.vanilla.css","source":"QGtleWZyYW1lcyBfMW84d2RzdzIgewogIGZyb20gewogICAgb3BhY2l0eTogMDsKICB9CiAgdG8gewogICAgb3BhY2l0eTogMTsKICB9Cn0KLl8xbzh3ZHN3MCB7CiAgd2lkdGg6IDEwMHZ3OwogIGhlaWdodDogMTAwdmg7CiAgcG9zaXRpb246IGFic29sdXRlOwogIHRvcDogMDsKICBsZWZ0OiAwOwogIHotaW5kZXg6IDA7Cn0KLl8xbzh3ZHN3MSB7CiAgcG9zaXRpb246IGFic29sdXRlOwogIHRvcDogMC40cmVtOwogIHJpZ2h0OiAwLjVyZW07CiAgZm9udC1zaXplOiAxcmVtOwogIG9wYWNpdHk6IDAuNDsKfQouXzFvOHdkc3czIHsKICBkaXNwbGF5OiBpbmxpbmUtZmxleDsKICBqdXN0aWZ5LWNvbnRlbnQ6IGNlbnRlcjsKICBhbGlnbi1pdGVtczogY2VudGVyOwogIGNvbG9yOiAjNGM1NjZhOwogIG1hcmdpbjogMC4ycmVtIDAuMnJlbSAwIDAuNXJlbTsKICBnYXA6IDAuNXJlbTsKICBhbmltYXRpb246IF8xbzh3ZHN3MiAycyBlYXNlLW91dCAxcyBib3RoOwogIHotaW5kZXg6IDEwMDsKfQouXzFvOHdkc3c0IHsKICBmb250LXNpemU6IDEuMnJlbTsKICBmb250LXdlaWdodDogMzAwOwogIHotaW5kZXg6IDA7Cn0="}!./node_modules/.pnpm/@vanilla-extract+webpack-plugin@2.2.0_webpack@5.75.0/node_modules/@vanilla-extract/webpack-plugin/extracted.js
./src/styles/home.css.ts

src/styles/home.css.ts.vanilla.css!=!./node_modules/.pnpm/@vanilla-extract+webpack-plugin@2.2.0_webpack@5.75.0/node_modules/@vanilla-extract/webpack-plugin/virtualFileLoader/dist/vanilla-extract-webpack-plugin-virtualFileLoader.cjs.js?{"fileName":"src/styles/home.css.ts.vanilla.css","source":"QGtleWZyYW1lcyBfMW84d2RzdzIgewogIGZyb20gewogICAgb3BhY2l0eTogMDsKICB9CiAgdG8gewogICAgb3BhY2l0eTogMTsKICB9Cn0KLl8xbzh3ZHN3MCB7CiAgd2lkdGg6IDEwMHZ3OwogIGhlaWdodDogMTAwdmg7CiAgcG9zaXRpb246IGFic29sdXRlOwogIHRvcDogMDsKICBsZWZ0OiAwOwogIHotaW5kZXg6IDA7Cn0KLl8xbzh3ZHN3MSB7CiAgcG9zaXRpb246IGFic29sdXRlOwogIHRvcDogMC40cmVtOwogIHJpZ2h0OiAwLjVyZW07CiAgZm9udC1zaXplOiAxcmVtOwogIG9wYWNpdHk6IDAuNDsKfQouXzFvOHdkc3czIHsKICBkaXNwbGF5OiBpbmxpbmUtZmxleDsKICBqdXN0aWZ5LWNvbnRlbnQ6IGNlbnRlcjsKICBhbGlnbi1pdGVtczogY2VudGVyOwogIGNvbG9yOiAjNGM1NjZhOwogIG1hcmdpbjogMC4ycmVtIDAuMnJlbSAwIDAuNXJlbTsKICBnYXA6IDAuNXJlbTsKICBhbmltYXRpb246IF8xbzh3ZHN3MiAycyBlYXNlLW91dCAxcyBib3RoOwogIHotaW5kZXg6IDEwMDsKfQouXzFvOHdkc3c0IHsKICBmb250LXNpemU6IDEuMnJlbTsKICBmb250LXdlaWdodDogMzAwOwogIHotaW5kZXg6IDA7Cn0="}!./node_modules/.pnpm/@vanilla-extract+webpack-plugin@2.2.0_webpack@5.75.0/node_modules/@vanilla-extract/webpack-plugin/extracted.js
TypeError: options.postcss is not a function
    at /home/runner/work/card.re-taro.dev/card.re-taro.dev/node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/build/webpack/loaders/postcss-loader/src/index.js:37:55
    at Span.traceAsyncFn (/home/runner/work/card.re-taro.dev/card.re-taro.dev/node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/trace/trace.js:79:26)
    at Object.loader (/home/runner/work/card.re-taro.dev/card.re-taro.dev/node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/build/webpack/loaders/postcss-loader/src/index.js:12:16)
    at tryRunOrWebpackError (/home/runner/work/card.re-taro.dev/card.re-taro.dev/node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/compiled/webpack/bundle5.js:28:308539)
    at __webpack_require_module__ (/home/runner/work/card.re-taro.dev/card.re-taro.dev/node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/compiled/webpack/bundle5.js:28:130357)
    at __nested_webpack_require_153265__ (/home/runner/work/card.re-taro.dev/card.re-taro.dev/node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/compiled/webpack/bundle5.js:28:129784)
    at /home/runner/work/card.re-taro.dev/card.re-taro.dev/node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/compiled/webpack/bundle5.js:28:130649
    at symbolIterator (/home/runner/work/card.re-taro.dev/card.re-taro.dev/node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/compiled/neo-async/async.js:1:14444)
    at done (/home/runner/work/card.re-taro.dev/card.re-taro.dev/node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/compiled/neo-async/async.js:1:14824)
    at Hook.eval [as callAsync] (eval at create (/home/runner/work/card.re-taro.dev/card.re-taro.dev/node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/compiled/webpack/bundle5.js:13:28771), <anonymous>:15:1)
    at /home/runner/work/card.re-taro.dev/card.re-taro.dev/node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/compiled/webpack/bundle5.js:28:129504
    at symbolIterator (/home/runner/work/card.re-taro.dev/card.re-taro.dev/node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/compiled/neo-async/async.js:1:14402)
    at timesSync (/home/runner/work/card.re-taro.dev/card.re-taro.dev/node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/compiled/neo-async/async.js:1:5027)
-- inner error --
TypeError: options.postcss is not a function
    at /home/runner/work/card.re-taro.dev/card.re-taro.dev/node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/build/webpack/loaders/postcss-loader/src/index.js:37:55
    at Span.traceAsyncFn (/home/runner/work/card.re-taro.dev/card.re-taro.dev/node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/trace/trace.js:79:26)
    at Object.loader (/home/runner/work/card.re-taro.dev/card.re-taro.dev/node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/build/webpack/loaders/postcss-loader/src/index.js:12:16)
    at Object.<anonymous> (/home/runner/work/card.re-taro.dev/card.re-taro.dev/node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/build/webpack/loaders/css-loader/src/index.js??ruleSet[1].rules[4].oneOf[0].use[1]!/home/runner/work/card.re-taro.dev/card.re-taro.dev/node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/build/webpack/loaders/postcss-loader/src/index.js??ruleSet[1].rules[4].oneOf[0].use[2]!/home/runner/work/card.re-taro.dev/card.re-taro.dev/node_modules/.pnpm/@vanilla-extract+webpack-plugin@2.2.0_webpack@5.75.0/node_modules/@vanilla-extract/webpack-plugin/virtualFileLoader/dist/vanilla-extract-webpack-plugin-virtualFileLoader.cjs.js?{"fileName":"src/styles/home.css.ts.vanilla.css","source":"QGtleWZyYW1lcyBfMW84d2RzdzIgewogIGZyb20gewogICAgb3BhY2l0eTogMDsKICB9CiAgdG8gewogICAgb3BhY2l0eTogMTsKICB9Cn0KLl8xbzh3ZHN3MCB7CiAgd2lkdGg6IDEwMHZ3OwogIGhlaWdodDogMTAwdmg7CiAgcG9zaXRpb246IGFic29sdXRlOwogIHRvcDogMDsKICBsZWZ0OiAwOwogIHotaW5kZXg6IDA7Cn0KLl8xbzh3ZHN3MSB7CiAgcG9zaXRpb246IGFic29sdXRlOwogIHRvcDogMC40cmVtOwogIHJpZ2h0OiAwLjVyZW07CiAgZm9udC1zaXplOiAxcmVtOwogIG9wYWNpdHk6IDAuNDsKfQouXzFvOHdkc3czIHsKICBkaXNwbGF5OiBpbmxpbmUtZmxleDsKICBqdXN0aWZ5LWNvbnRlbnQ6IGNlbnRlcjsKICBhbGlnbi1pdGVtczogY2VudGVyOwogIGNvbG9yOiAjNGM1NjZhOwogIG1hcmdpbjogMC4ycmVtIDAuMnJlbSAwIDAuNXJlbTsKICBnYXA6IDAuNXJlbTsKICBhbmltYXRpb246IF8xbzh3ZHN3MiAycyBlYXNlLW91dCAxcyBib3RoOwogIHotaW5kZXg6IDEwMDsKfQouXzFvOHdkc3c0IHsKICBmb250LXNpemU6IDEuMnJlbTsKICBmb250LXdlaWdodDogMzAwOwogIHotaW5kZXg6IDA7Cn0="}!/home/runner/work/card.re-taro.dev/card.re-taro.dev/node_modules/.pnpm/@vanilla-extract+webpack-plugin@2.2.0_webpack@5.75.0/node_modules/@vanilla-extract/webpack-plugin/extracted.js:1:7)
    at /home/runner/work/card.re-taro.dev/card.re-taro.dev/node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/compiled/webpack/bundle5.js:28:901184
    at Hook.eval [as call] (eval at create (/home/runner/work/card.re-taro.dev/card.re-taro.dev/node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/compiled/webpack/bundle5.js:13:28549), <anonymous>:7:1)
    at /home/runner/work/card.re-taro.dev/card.re-taro.dev/node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/compiled/webpack/bundle5.js:28:130390
    at tryRunOrWebpackError (/home/runner/work/card.re-taro.dev/card.re-taro.dev/node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/compiled/webpack/bundle5.js:28:308493)
    at __webpack_require_module__ (/home/runner/work/card.re-taro.dev/card.re-taro.dev/node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/compiled/webpack/bundle5.js:28:130357)
    at __nested_webpack_require_153265__ (/home/runner/work/card.re-taro.dev/card.re-taro.dev/node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/compiled/webpack/bundle5.js:28:129784)
    at /home/runner/work/card.re-taro.dev/card.re-taro.dev/node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/compiled/webpack/bundle5.js:28:130649
    at symbolIterator (/home/runner/work/card.re-taro.dev/card.re-taro.dev/node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/compiled/neo-async/async.js:1:14444)
    at done (/home/runner/work/card.re-taro.dev/card.re-taro.dev/node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/compiled/neo-async/async.js:1:14824)

Generated code for /home/runner/work/card.re-taro.dev/card.re-taro.dev/node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/build/webpack/loaders/css-loader/src/index.js??ruleSet[1].rules[4].oneOf[0].use[1]!/home/runner/work/card.re-taro.dev/card.re-taro.dev/node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/build/webpack/loaders/postcss-loader/src/index.js??ruleSet[1].rules[4].oneOf[0].use[2]!/home/runner/work/card.re-taro.dev/card.re-taro.dev/node_modules/.pnpm/@vanilla-extract+webpack-plugin@2.2.0_webpack@5.75.0/node_modules/@vanilla-extract/webpack-plugin/virtualFileLoader/dist/vanilla-extract-webpack-plugin-virtualFileLoader.cjs.js?{"fileName":"src/styles/home.css.ts.vanilla.css","source":"QGtleWZyYW1lcyBfMW84d2RzdzIgewogIGZyb20gewogICAgb3BhY2l0eTogMDsKICB9CiAgdG8gewogICAgb3BhY2l0eTogMTsKICB9Cn0KLl8xbzh3ZHN3MCB7CiAgd2lkdGg6IDEwMHZ3OwogIGhlaWdodDogMTAwdmg7CiAgcG9zaXRpb246IGFic29sdXRlOwogIHRvcDogMDsKICBsZWZ0OiAwOwogIHotaW5kZXg6IDA7Cn0KLl8xbzh3ZHN3MSB7CiAgcG9zaXRpb246IGFic29sdXRlOwogIHRvcDogMC40cmVtOwogIHJpZ2h0OiAwLjVyZW07CiAgZm9udC1zaXplOiAxcmVtOwogIG9wYWNpdHk6IDAuNDsKfQouXzFvOHdkc3czIHsKICBkaXNwbGF5OiBpbmxpbmUtZmxleDsKICBqdXN0aWZ5LWNvbnRlbnQ6IGNlbnRlcjsKICBhbGlnbi1pdGVtczogY2VudGVyOwogIGNvbG9yOiAjNGM1NjZhOwogIG1hcmdpbjogMC4ycmVtIDAuMnJlbSAwIDAuNXJlbTsKICBnYXA6IDAuNXJlbTsKICBhbmltYXRpb246IF8xbzh3ZHN3MiAycyBlYXNlLW91dCAxcyBib3RoOwogIHotaW5kZXg6IDEwMDsKfQouXzFvOHdkc3c0IHsKICBmb250LXNpemU6IDEuMnJlbTsKICBmb250LXdlaWdodDogMzAwOwogIHotaW5kZXg6IDA7Cn0="}!/home/runner/work/card.re-taro.dev/card.re-taro.dev/node_modules/.pnpm/@vanilla-extract+webpack-plugin@2.2.0_webpack@5.75.0/node_modules/@vanilla-extract/webpack-plugin/extracted.js

Import trace for requested module:
src/styles/home.css.ts.vanilla.css!=!./node_modules/.pnpm/@vanilla-extract+webpack-plugin@2.2.0_webpack@5.75.0/node_modules/@vanilla-extract/webpack-plugin/virtualFileLoader/dist/vanilla-extract-webpack-plugin-virtualFileLoader.cjs.js?{"fileName":"src/styles/home.css.ts.vanilla.css","source":"QGtleWZyYW1lcyBfMW84d2RzdzIgewogIGZyb20gewogICAgb3BhY2l0eTogMDsKICB9CiAgdG8gewogICAgb3BhY2l0eTogMTsKICB9Cn0KLl8xbzh3ZHN3MCB7CiAgd2lkdGg6IDEwMHZ3OwogIGhlaWdodDogMTAwdmg7CiAgcG9zaXRpb246IGFic29sdXRlOwogIHRvcDogMDsKICBsZWZ0OiAwOwogIHotaW5kZXg6IDA7Cn0KLl8xbzh3ZHN3MSB7CiAgcG9zaXRpb246IGFic29sdXRlOwogIHRvcDogMC40cmVtOwogIHJpZ2h0OiAwLjVyZW07CiAgZm9udC1zaXplOiAxcmVtOwogIG9wYWNpdHk6IDAuNDsKfQouXzFvOHdkc3czIHsKICBkaXNwbGF5OiBpbmxpbmUtZmxleDsKICBqdXN0aWZ5LWNvbnRlbnQ6IGNlbnRlcjsKICBhbGlnbi1pdGVtczogY2VudGVyOwogIGNvbG9yOiAjNGM1NjZhOwogIG1hcmdpbjogMC4ycmVtIDAuMnJlbSAwIDAuNXJlbTsKICBnYXA6IDAuNXJlbTsKICBhbmltYXRpb246IF8xbzh3ZHN3MiAycyBlYXNlLW91dCAxcyBib3RoOwogIHotaW5kZXg6IDEwMDsKfQouXzFvOHdkc3c0IHsKICBmb250LXNpemU6IDEuMnJlbTsKICBmb250LXdlaWdodDogMzAwOwogIHotaW5kZXg6IDA7Cn0="}!./node_modules/.pnpm/@vanilla-extract+webpack-plugin@2.2.0_webpack@5.75.0/node_modules/@vanilla-extract/webpack-plugin/extracted.js
./src/styles/home.css.ts


> Build failed because of webpack errors
 ELIFECYCLE  Command failed with exit code 1.
```

### Validations

- [X] Check that there isn't [already an issue](https://github.com/vanilla-extract-css/vanilla-extract/issues) that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by thevisioner** [comment]
I can also confirm that there is a problem with the latest versions of Next.js. I also tested canary releases after v13.1.6 and the problem occurs with v13.1.7-canary.8 and up.. v13.1.7-canary.7 still works fine.

In my case the error message is "Error: You forgot to add 'mini-css-extract-plugin' plugin (i.e. `{ plugins: [new MiniCssExtractPlugin()] }`)"

**Comment by rickdunkin** [comment]
Seems likely to be related to this: https://github.com/vercel/next.js/issues/46374

**Comment by i-bsd** [comment]
Same issue here. Is there a fix?

**Comment by re-taro** [comment]
It looks the issue is related to here

https://github.com/vercel/next.js/pull/45455

**Comment by luke-h1** [comment]
Looking into this further it seems that this problem is due to the `next-plugin` package which uses next internals that are prone to breakages since they're not public APIs (according to https://github.com/vercel/next.js/issues/46374#issuecomment-1446065787). 

```typescript
import { lazyPostCSS } from 'next/dist/build/webpack/config/blocks/css';
import { getGlobalCssLoader } from 'next/dist/build/webpack/config/blocks/css/loaders';
```

**Comment by shuding** [comment]
Since vanilla-extract-css uses Next.js internal APIs and we recently upgraded it, it's no longer working. I opened a PR in Next.js to refactor that API to be non-breaking but in the meantime you'll have to pass: `hasAppDir: nextConfig.experimental?.appDir` here:

https://github.com/vanilla-extract-css/vanilla-extract/blob/c59aee20079d66e8abe4ca23e0a2d673940139a8/packages/next-plugin/src/index.js#L40-L47

**Comment by graup** [comment]
Hi @shuding, thanks for helping us out! We discovered the `hasAppDir` fix also in #929, but it seems that additional changes (inside Next?) may be required. Could you have a look at that issue?

I applied your suggestion in a local patch and am still getting the same error `options.postcss is not a function`.

**Comment by shuding** [comment]
I'll take a look at #929! This issue requires a change on the Next.js side too, which was just released as canary so you might want to try `next@canary` with that change.

**Comment by AndriRafnR** [comment]
This appears to be resolved with nextjs version 13.2.3
